NitroxServer-Subnautica/Program.cs

```diff
@@ -262,7 +269,6 @@ private static void StopAndExitServer()
         {
             Log.Info("Exiting ...");
             Server.Instance.Stop();
-            Environment.Exit(0);
```

Comment:
Wouldn't this cause the cmd window (if you choose this option) to stay and not automatically close?

Reply:
It still automatically closes when clicking exit button on console window or CTRL+C. Though naming could be changed, will do.